Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'druid-tools' # for druid validation and druid-tree parsing
 gem 'moab-versioning' # work with Moab Objects
 
 group :development, :test do
+  gem 'rspec-rails', '~> 4.0'
```

Comment:
Moved because 

1. without this change, the default rake task winds up not invoking RSpec at all;
2. rspec-rails docs suggest it should be in the dev/test bundler group not just test

Reply:
thanks!  i had noticed that rspec wouldn't run by default for me locally when calling `rake`, but i never made time to figure out why, and would just `bundle exec rubocop && bundle exec rspec`.